app/scripts/proactive/view/BeautifiedModalAdapter.js

```diff
@@ -0,0 +1,86 @@
+define(
+    [
+        'jquery',
+        'backbone',
+        'pnotify',
+        'list',
+        'backboneFormsAdapter',
+        'StudioApp',
+        'bootstrap',
+    ],
+
+    function ($, Backbone, PNotify) {
+
+        "use strict";
+
+        // Extension of the bootstrap modal form to operate some cosmetic operations on the form.
```

Comment:
Did you make sure this doesn't removed the style we have already done for the form? (spacing between fields, button colors...)

Reply:
This operation (beautifyForm) is already applied to the left panel form (in class ViewWithProperties). It adds all necessary classes to the fields, implement tooltips, etc. The problem was this operation was not performed on the modal window when we want to edit selection scripts. This new class allows it.